MAUI/MauiAppBasic/MSALClient/PlatformConfig.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// Platform specific configuration.
+    /// </summary>
+    public class PlatformConfig
+    {
+        /// <summary>
+        /// Instance to store data
+        /// </summary>
+        public static PlatformConfig Instance { get; } = new PlatformConfig();

```

Comment:
No need to use singleton in a data class.

Reply:
This is configuration data. So, singleton was chosen.